sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1288,7 +1288,7 @@ object SQLConf {
       .doc("When true, make use of Apache Arrow for columnar data transfers. Currently available " +
         "for use with pyspark.sql.DataFrame.toPandas, " +
         "pyspark.sql.SparkSession.createDataFrame when its input is a Pandas DataFrame, " +
-        "and createDataFrame when its input is an R DataFrame. " +
+        "createDataFrame when its input is an R DataFrame, and gapply[Collect] in R." +
```

Comment:
I am going to fix the doc later when this R with Arrow optimization is finished.